prog/vnet/rekey_tunnel.rb

```diff
@@ -0,0 +1,111 @@
+# frozen_string_literal: true
+
+class Prog::Vnet::RekeyTunnel < Prog::Base
+  subject_is :nic
+
+  def start
+    ipsec_tunnel = nic.src_ipsec_tunnels.first
+    outs = ipsec_tunnel.cmd_src_nic("sudo ip -n #{ipsec_tunnel.vm_name(ipsec_tunnel.src_nic)} xfrm policy").split("\n")
```

Comment:
consider `ip --json` (`ip -j`)

Reply:
that doesn't work for xfrm, output doesn't change :(